### DISCUSSION TITLE
ToshinFoxのディスカッションへようこそ！

### DISCUSSION THREAD

**Discussion by tomo0611** [tomo0611/ToshinFox#1] [General]
## 👋 いらっしゃい !
  ディスカッションをあなたとつながる場所にしたいです！だから、
  * 質問して、どぞ
  * 提案を共有してね！
  * 他のコミュニティーメンバーを招待してね！
  * 心を開いて一緒に作っていこう！ 💪

  始めるにあたって、自己紹介してね！

![vlcsnap-2020-12-11-17h28m11s489](https://user-images.githubusercontent.com/31959526/101880444-4749ef80-3bd6-11eb-8864-866e5a7963ed.png)

**Comment by l9ic8kbuof** [comment]
おもしろいです
vodfilepathの数字はなにか規則性あるんでしょうか?

**Reply by tomo0611** [reply]
講義用のvodfilepathは基本的にランダムですね。

1分間ニュース(**1分間じゃない**)は`/GD/1minXXX/1.0/WV/800/1minXXX.mpd`(XXXは3桁の数字、2桁以下の場合は上位を0で埋める)で規則性があります。というよりは`OneMinNewsXXX.aspx`からXXXがわかります。
理事長挨拶や激励メッセージ、WEBホームルームもこの1分間ニュースを利用しています。

**Reply by tomo0611** [reply]
まあ万が一、vodfilepathを算出できてもWidevineで保護されているので見れないんですけどね
